snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: minio
-version: git
+version: master
```

Comment:
This is hardcoded, for the moment. All the snaps published will say that their version is "master". I made this change because I think it enables you to start testing in the edge channel. The snaps in the edge channel don't appear on user searches, and I hope that by the time you are ready to publish the stable snap, we would have solved the issue on our side and we can go back to get the version from the `git` tag.

Another approach would be to use `version-script`, that lets you run any command you want to generate the version, just making sure that the value returned is less than 32 chars until we solve this.